server/opendp_apps/dataverses/dataverse_download_handler.py

```diff
@@ -20,9 +20,6 @@
 
 class DataverseDownloadHandler(BasicErrCheck):
 
-    ERR_FAILED_TO_READ_DATASET = 'Failed to read the dataset.'
-    ERR_DATASET_POINTER_NOT_SET = 'In order to profile the data, the "dataset_pointer" must be set.'
-
     def __init__(self, dv_file_info: DataverseFileInfo, **kwargs):
```

Comment:
Please make sure there is only one space between functions in a class - this will drive the linter crazy